README.md

```diff
@@ -87,18 +87,11 @@ the project will install all dependencies and run `lerna bootstrap` to initializ
 With command
 
 ```bash
-$ yarn website-dev
+$ yarn docs:serve
```

Comment:
I think this should be `yarn docs:dev`

Reply:
Please address this comment before merged.

Reply:
Done.